roles/diyidp/templates/config-override.php.j2

```diff
@@ -0,0 +1,6 @@
+<?php
+$config['certdir'] = 'config/cert/';
+$config['metadatadir'] = 'config/metadata/';
+$config['auth.adminpassword'] = 'secret';
```

Comment:
Is this intentional?

Reply:
not at all, I missed this. Fixed by using a variable